bindings/pyroot_legacy/src/Converters.cxx

```diff
@@ -422,7 +422,7 @@ Bool_t PyROOT::TULongConverter::ToMemory( PyObject* value, void* address )
 }
 
 ////////////////////////////////////////////////////////////////////////////////
-/// construct python object from C++ unsigned int read at <address>
+/// construct python object from C++ unsigned int read at `<address>`
```

Comment:
Not too important since this is legacy PyROOT code, but if the intent is to use backticks around the name of a formal parameter, shouldn't we remove the `<>`?

Reply:
doxygen understands `<>` as a special command. With the backticks I was tryinig to 'escape it' without changing its appearance in the html docu. But yes, in principle we could remove instead the <> and write just ticks.

Reply:
Yes it would be better then to just have the backticks, thanks!

Reply:
Changed to only backticks ;)